taxonomies/brands.txt

```diff
@@ -238,3 +244,2492 @@ wikidata:en: Q10724505
 xx: Änglamark
 wikidata:en: Q65229572
 
+
+xx: indomie
+wikidata:en: Q27859
+
+xx: Mars
+wikidata:en: Q31305
+
+xx: Funyuns
+wikidata:en: Q49853
+
+xx: Whoppers
+wikidata:en: Q126392
+
+xx: Activia
+wikidata:en: Q176740
+
+xx: Kinder Surprise
+wikidata:en: Q181042
+
+xx: Nutella
+wikidata:en: Q212193
+
+xx: Toblerone
+wikidata:en: Q215707
+
+xx: Kit Kat
+wikidata:en: Q367251
+
+xx: After Eight
+wikidata:en: Q388973
+
+xx: Cracker Jack
+wikidata:en: Q455721
+
+xx: Kinder Happy Hippo
+wikidata:en: Q464501
+
+xx: Balisto
+wikidata:en: Q468828
+
+xx: Edy's Pie
+wikidata:en: Q477334
+
+xx: Ferrero Rocher
+wikidata:en: Q526516
+
+xx: Pringles
+wikidata:en: Q528187
+
+xx: Cheez Whiz
+wikidata:en: Q536691
+
+xx: McVitie's
+wikidata:en: Q557614
+
+xx: Ben's Original
+wikidata:en: Q566095
+
+xx: Tokyo Banana
+wikidata:en: Q575328
+
+xx: Calippo
+wikidata:en: Q595784
+
+xx: Old El Paso
+wikidata:en: Q600407
+
+xx: Peek Freans
+wikidata:en: Q699080
+
+xx: Lay’s
```

Comment:
I, for one, am in favour of using typographically correct punctuation, but taxonomy files are otherwise consistently using ASCII punctuation, so probably
```suggestion
xx: Lay's
```